File: dotnet-maui/ReservationsDotnetMaui/Services/ContextService.cs
namespace ReservationsDotnetMaui.Services;

public static class ContextService
{
    public static string? CurrentOffice { get; set; }
    public static string? CurrentUser { get; set; }

    public static string GetContextDisplay()
    {
        var parts = new List<string>();
        
        if (!string.IsNullOrEmpty(CurrentUser))
        {
            parts.Add($"User: {CurrentUser}");
        }
        
        if (!string.IsNullOrEmpty(CurrentOffice))
        {
            parts.Add($"Office: {CurrentOffice}");
        }
        
        return parts.Count > 0 ? string.Join(" | ", parts) : "No context";
    }
}

File: flutter/flutterapp/lib/services/context_service.dart
class ContextService {
  ContextService._();

  static String? currentOffice;
  static String? currentUser;

  static String getContextDisplay() {
    final parts = <String>[];
    final u = currentUser;
    if (u != null && u.isNotEmpty) {
      parts.add('User: $u');
    }
    final o = currentOffice;
    if (o != null && o.isNotEmpty) {
      parts.add('Office: $o');
    }
    return parts.isNotEmpty ? parts.join(' | ') : 'No context';
  }
}
